plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Browse.js

```diff
@@ -166,6 +171,16 @@ Ext.define('NX.coreui.controller.Browse', {
   },
 
   /**
+   * @overload
```

Comment:
no such tag, I think you mean @override but it also looks like we might be mis-using that... https://github.com/senchalabs/jsduck/wiki/@override